src/seismometer/_api.py

```diff
@@ -247,20 +252,19 @@ def generate_fairness_audit(
 
     if NotebookHost.supports_iframe() and fairness_path.exists():
         return load_as_iframe(fairness_path, height=height)
-
-    target = pdh.event_value(target_column)
     data = (
         pdh.event_score(
             sg.data(),
```

Comment:
With the transition of sg.output and target to XX_column vars, should this be fully transitioned to have sg.data passed in as well?  That would make this method unaware of sg.  

Fine to defer since I know bigger changes to this are coming

Reply:
Lets defer because I want to remove Aequitas, which will make me re-write this anyway.